### PR TITLE
documentation part XI (sample outputs)

### PR DESCRIPTION
**Pull request by tobes**
add sample output to modules

### CODE REVIEW

File: py3status/modules/clock.py [hunk 1]
```diff
@@ -81,6 +81,11 @@
 @author tobes
 @license BSD
 
+SAMPLE OUTPUT
+{'full_text': 'Sun 15 Jan 2017 23:27:17 GMT'}
+
+london
+{'full_text': 'Thursday Feb 23 1:42 AM London'}
```
Comment (lasers) [inline py3status/modules/clock.py]: No such time difference exists on our planet Earth.

File: py3status/modules/mpd_status.py [hunk 2]
```diff
@@ -51,6 +51,9 @@
 
 @author shadowprince, zopieux
 @license Eclipse Public License
+
+SAMPLE OUTPUT
+{'full_text': '[play] Music For Programming - Idol Eyes'}
```
Comment (lasers) [inline py3status/modules/mpd_status.py]:
```
{'color': '#00ff00', 'full_text': '[play] Music For Programming - Idol Eyes'}
{'color': '#ffff00', 'full_text': '[pause] Music For Programming - Idol Eyes'}
{'color': '#ff0000', 'full_text': '[stop] Music For Programming - Idol Eyes'}
```

File: py3status/modules/netdata.py [hunk 3]
```diff
@@ -24,6 +24,12 @@
     {total}    color threshold of total usage
 
 @author Shahin Azad <ishahinism at Gmail>
+
+SAMPLE OUTPUT
+[
+    {'color': '#FF0000', 'full_text': 'LAN(Kb):  16.9↓   5.2↑', 'separator': True},
+    {'color': '#FF0000', 'full_text': 'T(Mb): 2264↓ 143↑ 2407↕'}
```
Comment (lasers) [inline py3status/modules/netdata.py]: No more separator. We should use different color too. Throw `{nic}` in there too. I think this...
```
[
{'full_text': 'eno1'},
{'full_text': 'LAN(Kb): 671.9↓  31.4↑ ', 'color': '#00ff00'},
{'full_text': 'T(Mb): 14130↓ 702↑ 14832↕', 'color': '#ff0000'},
]
```

File: py3status/modules/spaceapi.py [hunk 4]
```diff
@@ -25,6 +25,9 @@
 
 @author timmszigat
 @license WTFPL <http://www.wtfpl.net/txt/copying/>
+
+SAMPLE OUTPUT
+{'color': '#FF0000', 'full_text': 'closed since 16:38'}
```
Comment (lasers) [inline py3status/modules/spaceapi.py]: And this. `{'color': '#00FF00', 'full_text': 'open since 05:41'}`

File: py3status/modules/timer.py [hunk 5]
```diff
@@ -20,6 +20,36 @@
     {timer} display hours:minutes:seconds
 
 @author tobes
+
+SAMPLE OUTPUT
+[
+    {'color': '#CCCCCC', 'full_text': 'Timer '},
```
Comment (lasers) [inline py3status/modules/timer.py]: We took out `#cccccc` for `Timer` text to create `format = "{timer}"`.

**Comment by lasers** [comment]
Missing: `keyboard_locks`. That's like... my module, man. 😡

File: py3status/modules/netdata.py (hunk 3 above)
Comment (tobes) [inline py3status/modules/netdata.py]: I'll wait till #846 is resolved

File: py3status/modules/clock.py (hunk 1 above)
Comment (tobes) [inline py3status/modules/clock.py]: please explain more

File: py3status/modules/bluetooth.py [hunk 6]
```diff
@@ -22,6 +22,12 @@
 
 @author jmdana <https://github.com/jmdana>
 @license GPLv3 <http://www.gnu.org/licenses/gpl-3.0.txt>
+
+SAMPLE OUTPUT
+{'color': '#00FF00', 'full_text': u'BT: Microsoft Bluetooth Notebook Mouse 5000'}
+
+off
+{'color': '#FF0000', 'full_text': u'BT: OFF'}
```
Comment (tobes) [inline py3status/modules/bluetooth.py]: waiting on #722

**Comment by tobes** [comment]
> Missing: keyboard_locks.

keeping the best till last :)

**Comment by ultrabug** [comment]
Huge work as usual, thanks a lot

**Pull request merged**